CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Contributing to EddieBotUI
+
+For **help**, **support**, and **questions** please use our [community chat](https://discord.com/invite/jZQs6Wu) 🚑.
+
+---
+
+## Where to Start
+
+Learn [how to contribute code on Github](https://codeburst.io/a-step-by-step-guide-to-making-your-first-github-contribution-5302260a2940).
+
+If you're a developer looking to contribute, but you're not sure where to begin, check out the [good first issue](https://github.com/EddieJaoudeCommunity/EddieBotUI/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) label on Github, which contains small tasks that have been specifically flagged as being friendly to new contributors.
+
+After that, if you're looking for something a little more challenging to sink your teeth into, there's a broader [help wanted](https://github.com/EddieJaoudeCommunity/EddieBotUI/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) label encompassing issues which need some love.
+
+Please, take a look [here](https://www.conventionalcommits.org/en/v1.0.0/) before commit. We follow Conventional Commits guidelines to keep our commits constistent.
+
+If you have a straightforward bug fix or improvement, feel free to contribute it in a [Pull Request](https://github.com/EddieJaoudeCommunity/EddieBotUI/pulls).
+
+If you have an idea for a new feature, please start by [searching the issues](https://github.com/EddieJaoudeCommunity/EddieBotUI/issues) to check that the feature has not already been suggested and then suggest it by [opening a new issue](https://github.com/EddieJaoudeCommunity/EddieBotUI/issues/new), as adding new features to EddieBotUI first requires some analysis around the design and spec.
+
+## Other ways to help
+
+If you're not a developer there are still plenty of ways that you can help. We always need help with:
+
+- Helping our EddieJaoudeCommunity on the [chat](https://discord.com/invite/jZQs6Wu)
+- Improving the [documentation](https://github.com/EddieJaoudeCommunity/EddieBotUI/tree/develop/docs) and writing tutorials
+- Testing and quality assurance
+- Promoting EddieJaoudeCommunity to others by blogging, vlogging, code streaming, talking etc.
```

Comment:
Here it is `EddieJaoudeCommunity` and other places it is 3 words. Not sure which is better. I think easier to read at 3 separate words in bold?

Reply:
I'll fix that ;)